engine/CMakeLists.txt

```diff
@@ -3,26 +3,35 @@
 
 option(BUILD_ENGINE_SAMPLES "Build cubos engine samples" OFF)
 
-message( "# Building engine samples: " ${BUILD_ENGINE_SAMPLES})
+message("# Building engine samples: " ${BUILD_ENGINE_SAMPLES})
 
 # Set engine source files
 
 set(CUBOS_ENGINE_SOURCE
     "src/cubos/engine/cubos.cpp"
-)
+
+    src/cubos/engine/rendering/renderer.cpp
+    src/cubos/engine/rendering/deferred/deferred_renderer.cpp
+    src/cubos/engine/rendering/post_processing/post_processing_pass.cpp
```

Comment:
Change `post_processing` to `pps` (post processing stack) and rename `post_processing_pass` to `pass`.